test/resources/controls/aws_parallelcluster_config/efa_system_settings_configured_spec.rb

```diff
@@ -0,0 +1,9 @@
+control 'efa_system_settings_configured' do
```

Comment:
Nitpick. Can we name the file `efa_spec.rb` to make it more generic?

Reply:
Done